activities/Measure.activity/locale.ini

```diff
@@ -0,0 +1,537 @@
+TutoShowHarmonicsContent=Vous pouvez afficher ou masquer les harmoniques en appuyant sur ce bouton. Les lignes d'harmoniques seront affichées uniquement si un instrument est choisi et que le mode fréquence est actif. Les lignes sont de la même couleur que les notes.
+TutoTuningFreqInputTitle=Saisie de la fréquence
+TutoTuningFreqInputContent=Cette valeur affiche la combinaison de la note et l'octave quand aucun instrument n'est choisi. Vous pouvez aussi directement entrer la valeur de la fréquence dans le champ.
+TutoTuningLineTitle=Voir ou cacher ligne de la gamme
+TutoTuningLineContent=La ligne de la gamme sera affichée à la même fréquence que la valeur du champs de saisie. Vous pouvez voir ou cacher la ligne en appuyant sur ce bouton. En même temps que la ligne de la gamme, la note et l'octave correspondant à la fréquence la plus haute sera également affichée dans le coin en haut à droite de l'écran.
+TutoPlayStopNoteTitle=Jouer ou arrêter note
+TutoPlayStopNoteContent=Vous pouvez jouer our arrêter la note en appuyant sur ce bouton. La note jouée sera celle correspondant à la fréquence entrée dans le champ de saisie.
+TutoCaptureWaveformTitle=Capture onde
+TutoCaptureWaveformContent=Vous pouvez faire une capture d'écran de l'onde en appuyant sur ce bouton. La capture sera enregistrée dans le journal.
+
+[es]
+SecondsAgo=Segundos atrás
+Ago={{time}} atrás
+Minutes_one=minuto
+Minutes_other=minutos
+Hours_one=hora
+Hours_other=horas
+Days_one=día
+Days_other=días
+Weeks_one=semana
+Weeks_other=semanas
+Months_one=mes
+Months_other=meses
+Years_one=año
+Years_other=años
+Shutdown=Apagar
+Restart=Reiniciar
+MySettings=Mis ajustes
+SearchHome=Buscar en hogar
+JournalEmpty=Tu diario está vacío
+NoMatchingActivities=No hay actividades coincidentes
+ClearSearch=Limpiar búsqueda
+SearchJournal=Buscar en el diario
+Anytime=Cualquier momento
+Today=Hoy
+SinceYesterday=Desde ayer
+PastWeek=La semana pasada
+PastMonth=El mes pasado
+PastYear=El año pasado
+UserJoined={{name}} joined
+UserLeft={{name}} left
+A=la
+B=si
+C=do
+D=re
+E=mi
+F=fa
+G=sol
+A♯/B♭=la♯/si♭
+C♯/D♭=do♯/re♭
+D♯/E♭=re♯/mi♭
+F♯/G♭=fa♯/sol♭
+G♯/A♭=sol♯/la♭
+
+[pt]
+SecondsAgo=Segundos atrás
+Ago={{time}} atrás
+Minutes_one=minuto
+Minutes_other=minutos
+Hours_one=hora
+Hours_other=horas
+Days_one=dia
+Days_other=dias
+Weeks_one=semana
+Weeks_other=semanas
+Months_one=mês
+Months_other=meses
+Years_one=ano
+Years_other=anos
+Shutdown=Desligar
+Restart=Reiniciar
+MySettings=Minhas Configurações
+SearchHome=Pesquisar na minha página
+JournalEmpty=Seu diário está vazio
+NoMatchingActivities=Sem atividades correspondentes
+ClearSearch=Limpar pesquisa
+SearchJournal=Pesquisar no diário
+Anytime=A qualquer momento
+Today=Hoje
+SinceYesterday=Desde ontem
+PastWeek=Semana passada
+PastMonth=Mês passado
+PastYear=Ano passado
+UserJoined={{name}} ingressou
+UserLeft={{name}} saiu
+A=lá
+B=si
+C=dó
+D=ré
+E=mi
+F=fá
+G=sol
+A♯/B♭=lá♯/si♭
+C♯/D♭=dó♯/ré♭
+D♯/E♭=ré♯/mi♭
+F♯/G♭=fá♯/sol♭
+G♯/A♭=sol♯/lá♭
```

Comment:
Why you need to keep the .ini file?

Reply:
No need, i just thought it's inappropriate to delete the file. Should I remove the file?

Reply:
Yes. It's now useless.

Reply:
Okay. I will edit my last commit.